<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSpaceWidget</class>
 <widget class="QWidget" name="ColorSpaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_colorSpacePreset">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_colorSpacePreset">
       <property name="toolTip">
        <string>Select ColorSpace Preset</string>
       </property>
       <item>
        <property name="text">
         <string>sRGB - HDTV (ITU-R BT.709-5)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ROMM RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Adobe RGB 98</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Apple RGB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NTSC (FCC 1953)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NTSC (1979) (SMPTE C/-RP 145)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PAL/SECAM (EBU 3213)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CIE (1931) E</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTabWidget" name="tabs_colorSpaceTabs">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_whitepoint">
        <attribute name="title">
         <string>Whitepoint</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_15">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_28">
           <item row="0" column="0">
            <widget class="QLabel" name="label_whitePointPreset">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Preset</string>
             </property>
             <property name="buddy">
              <cstring>comboBox_whitePointPreset</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QComboBox" name="comboBox_whitePointPreset">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>A</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>B</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>C</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>D50</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>D55</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>D65</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>D75</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>E</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>F2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>F7</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>F11</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_whitePointX">
             <property name="text">
              <string>White X</string>
             </property>
             <property name="buddy">
              <cstring>slider_whitePointX</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="slider_whitePointX">
             <property name="toolTip">
              <string>Adjust Whitepoint X</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QDoubleSpinBox" name="spinBox_whitePointX">
             <property name="toolTip">
              <string>White X</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_whtePointY">
             <property name="text">
              <string>White Y</string>
             </property>
             <property name="buddy">
              <cstring>slider_whitePointY</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSlider" name="slider_whitePointY">
             <property name="toolTip">
              <string>Adjust Whitepoint Y</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QDoubleSpinBox" name="spinBox_whitePointY">
             <property name="toolTip">
              <string>White Y</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_rgb">
        <attribute name="title">
         <string>RGB</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_rgbXY">
             <property name="text">
              <string>Red/Green/Blue XY</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSlider" name="slider_redX">
             <property name="toolTip">
              <string>Red X</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="spinBox_redX">
             <property name="toolTip">
              <string>Red X</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="slider_redY">
             <property name="toolTip">
              <string>Red Y</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QDoubleSpinBox" name="spinBox_redY">
             <property name="toolTip">
              <string>Red Y</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSlider" name="slider_greenX">
             <property name="toolTip">
              <string>Green X</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="spinBox_greenX">
             <property name="toolTip">
              <string>Green X</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSlider" name="slider_greenY">
             <property name="toolTip">
              <string>Green Y</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QDoubleSpinBox" name="spinBox_greenY">
             <property name="toolTip">
              <string>Green Y</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QSlider" name="slider_blueX">
             <property name="toolTip">
              <string>Blue X</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="spinBox_blueX">
             <property name="toolTip">
              <string>Blue X</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSlider" name="slider_blueY">
             <property name="toolTip">
              <string>Blue Y</string>
             </property>
             <property name="maximum">
              <number>512</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QDoubleSpinBox" name="spinBox_blueY">
             <property name="toolTip">
              <string>Blue Y</string>
             </property>
             <property name="keyboardTracking">
              <bool>false</bool>
             </property>
             <property name="maximum">
              <double>1.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
